ros2/naive/include/naive/incrementer.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+#include <memory>
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int64.hpp"
+
+class Incrementer {
+public:
+	Incrementer(std::shared_ptr<rclcpp::Node> node, const std::string& in_topic, const std::string& out_topic);
+private:
+	static constexpr int QUEUE_SIZE = 10;
```

Comment:
Let's move this to a anon namespace in `incrementer.cpp` as in
```
#include "std_msgs/msg/int64.hpp"

namespace {
constexpr auto queue_size = 10;
}
Incrementer::Incrementer(...
```

Reply:
Done